apps/quarkus-json/src/main/java/org/acme/ExampleResource.java

```diff
@@ -0,0 +1,220 @@
+package org.acme;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
```

Comment:
Why would this not need the `jakarta` import changes similar to what is being done in  `quarkus-3.patch`?

Reply:
@jerboaa Very good catch.  The app is used only in PerfCheck tests and those are not on by default, so the CI here didn't catch it.  ...and my Jenkins job doesn't perf tests Q 3.x yet.
I will add PerfCheck tests to the CI in this repo so as everything is covered.

Lemme patch it.

Reply:
Thanks!